.travis/shellchecker

```diff
@@ -75,9 +75,10 @@ main () {
 default () {
     # run a strict check on all "functional" scripts
     main '.' \
-        --exclude 'tests/' \
         --exclude 'cylc/flow/tests/parsec' \
         --exclude 'etc/bin/live-graph-movie.sh' \
+        --exclude 'flakytests/' \
+        --exclude 'tests/' \
```

Comment:
Looks like I have left the exclusion in. It should no longer be necessary for `flakytests/`.